<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>751</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>20</horstretch>
    <verstretch>20</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>20</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>765</width>
    <height>1235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PwnBrowser</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/appicon.png</normaloff>icons/appicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>1051</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="btn1">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Google Chrome</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/chrome.png</normaloff>icons/chrome.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mozilla Firefox</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/firefox.png</normaloff>icons/firefox.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Microsoft Edge</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/edge.png</normaloff>icons/edge.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn4">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Opera browser</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/opera.png</normaloff>icons/opera.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btn5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brave browser</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/brave.png</normaloff>icons/brave.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="logins">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="crack">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Book Antiqua</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CRACK</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>1051</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="profile_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="key_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="key">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="selec_br">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Browser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="default_locations">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Default</string>
    </property>
   </widget>
   <widget class="QLabel" name="results">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Verbose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="info_btn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>1001</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>1051</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1049</width>
       <height>379</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>logins</zorder>
   <zorder>crack</zorder>
   <zorder>progressBar</zorder>
   <zorder>profile_btn</zorder>
   <zorder>key_btn</zorder>
   <zorder>key</zorder>
   <zorder>selec_br</zorder>
   <zorder>default_locations</zorder>
   <zorder>results</zorder>
   <zorder>info_btn</zorder>
   <zorder>scrollArea</zorder>
   <zorder>output</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
